composer.json

```diff
@@ -28,6 +33,16 @@
         }
     },
     "require-dev": {
-        "sllh/php-cs-fixer-styleci-bridge": "^2.1"
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6",
+        "php-http/guzzle6-adapter": "^1.1",
+        "php-http/message": "^1.5",
+        "phpunit/phpunit": "^5.2",
```

Comment:
Try to use phpunit globally.

BTW, the last version is v6: https://phpunit.de/